docs/src/content/docs/resources/glossary.mdx

```diff
@@ -8,6 +8,14 @@ import { LinkOut } from '@interledger/docs-design-system'
 
 An account servicing entity provides and maintains a payment account for a sender and recipient, and is a regulated entity in the country/countries it operates.
 
+## Asset code
+
+An `assetCode` is a code representation of the underlying asset used to make a payment. The `assetCode` should be an <LinkOut href='https://en.wikipedia.org/wiki/ISO_4217'>ISO 4217</LinkOut> currency code. The ISO 4217 currency code representing the US Dollar is USD.
```

Comment:
This should be updated as well (based on the discussions I've seen in the pull request), since the asset code might be something outside of ISO4217.